backend/core/src/main/java/org/sonarsource/sonarlint/core/analysis/AnalysisService.java

```diff
@@ -152,6 +154,8 @@ public class AnalysisService {
   private final OpenFilesRepository openFilesRepository;
   private final ClientFileSystemService clientFileSystemService;
   private boolean automaticAnalysisEnabled;
+  private final ScheduledExecutorService scheduledAnalysisExecutor =
+    Executors.newSingleThreadScheduledExecutor(r -> new Thread(r, "SonarLint Analysis Executor"));
```

Comment:
You still need to close this executor when the service is shutdown

Reply:
Sorry, fixed now.